CHANGELOG.md

```diff
@@ -48,6 +49,8 @@
 - Remove default features. You need to enable required features explicitly now. As a quick workaround, you may want to use the
   new `full` feature which activates all features. See [PR 2918].
 
+
+- New `libp2p-onion` crate. See [PR 2899].
```

Comment:
This needs to go into a new section of the changelog :)

Reply:
I moved it, I thought it would make it into the 0.49.0 release. Didn't expect the delays.